packages/fx-core/tests/core/FxCore.test.ts

```diff
@@ -371,10 +371,47 @@ describe("Core basic APIs", () => {
     }
   });
 
+  it("addSso method should exist", async () => {
+    const restore = mockedEnv({
+      TEAMSFX_V3: "true",
+      TEAMSFX_DEBUG_TEMPLATE: "true", // workaround test failures when template changed but not release to GitHub alpha template
```

Comment:
@SLdragon @hund030 I added these 2 env to workaround test failures when template changed but not release to GitHub alpha template. Please see whether this looks good to you.

Reply:
Looks good~